pkg/utils/configs/config.go

```diff
@@ -38,6 +38,7 @@ connector {
     name: {{.Name}}-connector
     host: {{.Host}}
     port: {{.Port}}
+    dataConnectionCount: {{.Dcc}}
```

Comment:
We should only set this if it is non zero. That way we have a way to revert to the router default if desired.